app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
```

Comment:
- [ ] I failed to register because the app was asking me to confirm my email address, and unfortunately, I didn't receive any email for me to confirm, note: I used my actual (real) email. This prevented me from testing the app.

![email](https://user-images.githubusercontent.com/53124548/196015242-689c4897-0a0f-4093-864b-434da1fd7542.png)

Reply:
Hi @elisha2kyakpo1,

As this application is not in production yet, it's not set up to actually send emails. So, to register a new account and be able to test the application, you can check the terminal where you are running the server. The message the user is supposed to receive by email will be printed there, so you just need to click on the confirmation link:

![terminal](https://user-images.githubusercontent.com/64566209/196016197-83a533c8-8c5d-49b7-9904-7174d2d241d3.png)

Reply:
I also configured the application to send a fake smtp email to mailcatcher. So, alternatively, you could install the [MailCatcher](https://mailcatcher.me/) program with the command:
```
$ gem install mailcatcher
```
Doing this you will be able to catch the emails sent by this application on a more friendly interface:
![mailcatcher](https://user-images.githubusercontent.com/64566209/196016406-20e7df78-1dfb-4951-82de-aca653371ba9.png)